src/commands/analytics/log-flow.ts

```diff
@@ -0,0 +1,147 @@
+        debug(`Failed to load the logs - ${inspect(error)}`);
+        throw failure(ErrorCodes.Exception, "failed to load the logs");
+      }
+    }, (response: ClientResponse<models.LogContainer>, responsesProcessed: number) => {
+      // processing http response
+      const result = response.result;
+      if (result.logs.length) {
+        // new logs were received
+        options = { start: result.lastReceivedLogTimestamp };
+
+        // take no more than specified number of logs from the first request response
+        const filteredLogs = responsesProcessed ? this.filterLogs(result.logs, this.installationId) : 
+          _.takeRight(this.filterLogs(result.logs, this.installationId), logsCount);
+        for (const logEntry of filteredLogs) {
+          this.showLogEntry(streamingOutput, logEntry);
+        }
+      }
+    }, this.showContinuously, ShowLogFlowCommand.delayBetweenRequests, "Loading logs...");
+    
+    streamingOutput.finish();
+
+    return success();
+  }
+
+  private validateParameters(): void {
+    if (!_.isNil(this.logsCount)) {
+      const parsedNumberOfLogs = Number(this.logsCount);
+
+      if (!Number.isSafeInteger(parsedNumberOfLogs) || parsedNumberOfLogs < 1 || parsedNumberOfLogs > 100) {
+        throw failure(ErrorCodes.InvalidParameter, `${this.logsCount} is not a valid number of logs to show`);
+      }
+    }
+
+    if (!_.isNil(this.installationId)) {
+      if (!/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/g.test(this.installationId)) {
+        throw failure(ErrorCodes.InvalidParameter, `${this.installationId} is not a valid installation identifier`);
+      }
+    }
+  }
+
+  private filterLogs(logs: models.Log[], installId: string): models.Log[] {
+    if (!_.isNil(installId)) {
+      return logs.filter((logEntry) => logEntry.installId === installId);
+    } else {
+      return logs;
+    }
+  }
+
+  private showLogEntry(output: StreamingArrayOutput, logEntry: models.Log): void {
+    // setting common properties
+    let logString: string = `${logEntry.timestamp}, ${logEntry.installId}, ${logEntry.type}`;
+    let jsonObject = _({
+      date: logEntry.timestamp,
+      installId: logEntry.installId,
+      logType: logEntry.type
+    });
+
+    // adding log id
+    if (isEventLog(logEntry)) {
+      // event name for event log
+      logString += ", " + logEntry.name;
+      jsonObject.extend({ logId: logEntry.name});
+    } else if (isILogWithSessionId(logEntry)) {
+      // session id for logs with such property
+      logString += ", " + logEntry.sessionId;
+      jsonObject.extend({ logId: logEntry.sessionId});
+    }
+
+    // adding properties
+    if (isLogWithProperties(logEntry)) {
+      if (_.size(logEntry.properties)) {
+        logString += ", " + _.toPairs(logEntry.properties).map((pair) => pair.join(": ")).join(", ");
+      }
+      jsonObject.extend({ properties: logEntry.properties});
+    }
+
+    output.text(_.constant(logString), jsonObject);
```

Comment:
No, `_.constant(logString)` is equal to `() => logString`. I have changed it to the latter.